### DISCUSSION TITLE
Student Pack transfer

### DISCUSSION THREAD

**Discussion by GijsMdW** [community/community#23669] [GitHub Education]
<p>Hey,</p>
<p>I’d like to use my Student Pack on my personal account combined with my school’s e-mail address as second address instead of on a separate account. I applied for a Student Pack, but I didn’t get approved. Will deleting the seperate account make it possible to use the Student Pack on the other account?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>If you already have an approved student discount, you cannot be approved for another one. If you want to keep only one account, please <strong>keep the account with the student pack</strong> and delete the other account. You can then add your other email address to the account that has the student pack on it.</p>
<p>If you delete the account with the Student Pack, you will not be approved with the same email address on a different account.</p>

**Reply by klphumisithikul** [reply]
Is it possible to combine two github accounts (i.e., one forgotten personal and another personal)?

**Reply by sagar-io** [reply]
**Yes**, It is possible to combine(merge) two GitHub accounts.  For the clear documentation please visit:  [Merge Your Account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts)

**Comment by dhruv-tank-2910** [comment]
Greetings for the day
Dhruv Tank here,

Respected Sir/ma'am since I have got my student account on the github education around 1.5 years ago but 6 months ago I lost my phone and got the new phone after some time. Due to that I lost my 2FA approvl app also with the pass key for it and as the solution part of it I deleted the account and reCreated the account from the same E-mail address.
Now when I am trying to get the education pack verification pack it is showing me the error for that it is linked with the old account.
So can you please once check them So I can get the student education access back.
Thank You for the help and providing the pack for me as the student it would be the great help.

**Reply by bforero** [reply]
Heeelp, did you get any reply?
